authui/src/authflowv2/components/divider.css

```diff
@@ -0,0 +1,36 @@
+@layer components {
+  :root {
+    --divider__bg-color: var(--color-grey-300);
+    --divider__text-color: var(--color-grey-300);
+    --divider__font-family: var(--typography-body-large__font-family);
+    --divider__font-size: var(--typography-body-large__font-size);
+    --divider__line-height: var(--typography-body-large__line-height);
+    --divider__letter-spacing: var(--typography-body-large__letter-spacing);
+    --divider__font-weight: var(--typography-body-large__font-weight);
+  }
+
+  :root.dark {
+    /* TODO(davis): Dark mode styles */
+  }
+
+  .divider {
+    @apply absolute w-full h-px -z-10 opacity-30;
+
+    background-color: var(--divider__bg-color);
+  }
+
+  .divider__container {
+    @apply flex justify-center items-center relative;
+  }
+
+  .divider__text-container {
+    @apply bg-white text-center w-11;
```

Comment:
1. Instead of using fixed width, give it a padding so that it won't broken when the text length changed.

2. The background should be transparent because the background color might not be white. It is customizable. The element structure can be `<divider><OR><divider>` so we don't need to specify a background color.